battle-engine.js

```diff
@@ -541,6 +541,18 @@ BattlePokemon = (function () {
 				target = this.battle.runEvent('RedirectTarget', this, this, move, target);
 			}
 			targets = [target];
+
+			// Resolve apparent targets for Pressure.
+			if (move.pressureTarget) {
```

Comment:
Wait, is it really just this? Seems more straighforward to just `switch (move.pressureTarget || move.target)`?